README.md

```diff
@@ -51,7 +50,7 @@ terminus secret:set <site> file.json "{}" --type=file
 ```
 
 ```
-terminus secret:set <site> foo bar --scope=ic,user
+terminus secret:set <site> foo bar --scope=ic --scope=user
```

Comment:
I believe RyanW and TomS worked on changing this syntax so it's comma delimited rather than multiple `--scope` parameters, but I'm not sure if those changes have been merged yet or not..

Reply:
The [PR in question](https://github.com/pantheon-systems/terminus-secrets-manager-plugin/pull/16) was merged... I think it means it has to be comma delimited now.. requested confirmation via Slack..

Reply:
```suggestion
terminus secret:set <site> foo bar --scope=user,ic
```